{"name": "test_004", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 0", "trace": "self = <cases.testLogin.TestLogin object at 0x000001D8AB9636A0>\n\n    @allure.story(\"被测场景\")\n    # @allure.MASTER_HELPER.severity(\"blocker\")\n    @allure.severity(\"blocker\")\n    # @allure.MASTER_HELPER.step(\"断言结果\")\n    @allure.step(\"断言结果\")\n    @pytest.mark.dependence(name=\"test_004\")\n    def test_004(self):\n>       assert 1==0\nE       assert 1 == 0\n\ntestLogin.py:34: AssertionError"}, "steps": [{"name": "断言结果", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 0\n", "trace": "  File \"D:\\software\\PycharmProject\\appDemo\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\software\\PycharmProject\\appDemo\\cases\\testLogin.py\", line 34, in test_004\n    assert 1==0\n"}, "start": 1616999049017, "stop": 1616999049017}], "start": 1616999049017, "stop": 1616999049018, "uuid": "5ea75dfc-ca9a-4ba7-b5df-b827532a10c1", "historyId": "0c6207c64428924afd946da592053d4a", "testCaseId": "fc5e09bb0108f2913ab76e5c607a2a8c", "fullName": "cases.testLogin.TestLogin#test_004", "labels": [{"name": "feature", "value": "测试TestLogin"}, {"name": "severity", "value": "blocker"}, {"name": "story", "value": "被测场景"}, {"name": "tag", "value": "dependence(name='test_004')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "testLogin"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-MI5M7JM"}, {"name": "thread", "value": "15600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.testLogin"}]}